docs/pages/desktop-access/active-directory-manual.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Desktop Access Manual Setup
-description: Manually Connect Teleport to Active Directory and log into a Windows Desktop
+title: Desktop Access with Active Directory (Manual)
+description: Manually Connect Teleport to an Active Directory domain
 videoBanner: YvMqgcq0MTQ
 ---
```

Comment:
Should this link to the other Active Directory guide so the user knows when to use each one?